packages/file-adapters/lib/local-file.js

```diff
@@ -3,19 +3,19 @@ const path = require('path');
 const mkdirp = require('mkdirp');
 
 module.exports = class LocalFileAdapter {
-  constructor({ directory, route }) {
-    this.directory = path.resolve(directory);
-    this.route = route;
+  constructor({ src, path }) {
```

Comment:
`path` will be shadowed if you do it like this, so you need to either reassign it to a different variable or don't destructure:

```suggestion
  constructor({ src, path: inputPath }) {
```